Dockerfile

```diff
@@ -1,5 +1,6 @@
 # Build the manager binary
-FROM registry.access.redhat.com/ubi9/go-toolset:1.20.12 as builder
+
+FROM golang:1.22.0 as builder
```

Comment:
this needs to change @valerymo

Reply:
Thank you Michal. Changed to `mirror.gcr.io/library/golang:1.22.8` as discussed.